### PR TITLE
Ported TFLM Micro Speech example to CEVA-DSP SP500

### PR DESCRIPTION
**Pull request by yair-ehrenwald**
This is a port of the Micro Speech example to CEVA-DSP SP500 processor.

### CODE REVIEW

**Comment by gbaned** [comment]
@yair-ehrenwald Can you please resolve conflicts? Thanks!

**Comment by yair-ehrenwald** [comment]
> @yair-ehrenwald Can you please resolve conflicts? Thanks!

Done :)

File: tensorflow/lite/micro/examples/micro_speech/ceva/audio_provider.cc [hunk 1]
```diff
@@ -60,39 +60,48 @@ TfLiteStatus GetAudioSamples(tflite::ErrorReporter* error_reporter,
...
-    }
-  }
+
+void read_samples()
+{
+	int i =0;
+	uint8_t mem[3];
+	bool done;
+	for (int i = 0; i < kNoOfSamples; i++)
+	{
```
Comment (advaitjain) [inline tensorflow/lite/micro/examples/micro_speech/ceva/audio_provider.cc:91]: this seems to be counter to the formatting that we normally have.

Can you confirm that you're using -style=google with clang-format?

```
clang-format -i -style=google audio_provider.cc
```

**Review by advaitjain — CHANGES_REQUESTED** [review]

**Review by advaitjain — CHANGES_REQUESTED** [review]
also, it might be worth investigating what in your workflow is making the files executables.

This pull request has made main_functions.cc and executable and when I looked, ceva/audio_provider.cc is also an executable on master.

File: tensorflow/lite/micro/tools/make/targets/ceva_SP500_makefile.inc [hunk 2]
```diff
@@ -0,0 +1,45 @@
+
```
[the remark this answers is not in the document]
Comment (advaitjain) [reply tensorflow/lite/micro/tools/make/targets/ceva_SP500_makefile.inc:1]: @yair-ehrenwald, any thoughts on the feasibility of this?

[the remark this answers is not in the document]
Comment (yair-ehrenwald) [reply tensorflow/lite/micro/tools/make/targets/ceva_SP500_makefile.inc:1]: @advaitjain I don't see a reason why not, I'll update tomorrow, thanks :)

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
@yair-ehrenwald  Any update on this PR? Please. Thanks!

**Comment by gbaned** [comment]
@yair-ehrenwald Any update on this PR? Please. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 29 days that this pull-request has stalled. Please create a new pull-request with the requested changes.

**Pull request closed without merging**